releasewarrior/cli.py

```diff
@@ -296,3 +296,12 @@ def status(verbose, logger=LOGGER, config=CONFIG):
         for release in complete_releases:
             log_release_status(release, logger)
     ###
+
+@cli.command()
+@click.option('--push', is_flag=True, help="Pushes the data repo to upstream")
+@click.option('--diff', is_flag=True, help="diff against upstream")
+def data(push, diff, logger=LOGGER, config=CONFIG):
+    if push:
+        git.push(logger, config)
+    elif diff:
```

Comment:
perhaps we remove the diff feature until it is implemented. Or at least log that this feature doesn't work yet and remove the pdb line.